src/client/dfuse/ops/open.c

```diff
@@ -69,6 +69,9 @@ dfuse_cb_open(fuse_req_t req, fuse_ino_t ino, struct fuse_file_info *fi)
 
 	LOG_FLAGS(ie, fi->flags);
 
+	if (fi->flags & O_TRUNC)
```

Comment:
maybe we should handle this in DFS rather than dfuse?

Reply:
I just said that to Johann.

dfuse handles open of existing files by looking up the object and calling dfs_dup() so I think we'll still need this change but we probably do want something in dfs_open()

Reply:
FYI, it fixed the problem for me.

Reply:
well we have the flags parameter in both lookup and dup. so we could do the truncate there too.

Reply:
I think I agree with @mchaarawi, since this would centralize the logic in DFS for other applications to work the same way.

Reply:
Fine with me.